src/instana/instrumentation/redis.py

```diff
@@ -2,94 +2,115 @@
 # (c) Copyright Instana Inc. 2018
 
 
+from typing import Any, Callable, Dict, Tuple
 import wrapt
 
-from ..log import logger
-from ..util.traceutils import get_tracer_tuple, tracing_is_off
-
+from instana.log import logger
+from instana.span.span import InstanaSpan
+from instana.util.traceutils import get_tracer_tuple, tracing_is_off
 
 try:
     import redis
 
     EXCLUDED_PARENT_SPANS = ["redis", "celery-client", "celery-worker"]
 
-    def collect_tags(span, instance, args, kwargs):
+    def collect_tags(
+        span: InstanaSpan,
+        instance: redis.client.Redis,
+        args: Tuple[Any, ...],
```

Comment:
Can we be more specific of the type of items in the tuple? If its difficult to assess then please change it to below
```suggestion
        args: Tuple[object],
```